### DISCUSSION TITLE
Errore configurazione accesso remoto con Cloudflare

### DISCUSSION THREAD

**Discussion by diegoves** [indomus/forum#3108] [HUB: Home Assistant]
Ciao a tutti! sto seguendo la guida con Cloudflare per l’accesso da remoto ad HA, fatto i primi passi ma quando lancio il comando per creare la regola di routing ricevo questo errore (per il dominio sto usando godaddy):
 
`Failed to add route: code: 1003, reason: Failed to create record XXXXXX with err An A, AAAA, or CNAME record with that host already exists. For more details, refer to <https://developers.cloudflare.com/dns/manage-dns-records/troubleshooting/records-with-same-name/>.`

Qualcuno ha qualche idea di cosa sto sbagliando? la sostituzione dei nomi su godaddy è andata a buon fine 🤔

**Comment by snakuzzo** [comment]
Con qualche tua prova hai già creato il CNAME o RECORD (o quel che sia) per il quale stai tentando di configurare il tunnel.
Vai sulla dashboard di Cloudflare e cancellalo.
Consiglio...ti conviene creare un CNAME specifico per l'applicazione che stai tentando di espoore.
Es. Se hai comprato il dominio `pippo.it`, magari prova a creare il CNAME (che corrisponde al nome da utilizzare per il tunnel) con `ha.pippo.it`.

**Reply by diegoves** [reply]
Grazie mille per la risposta! Attualmente sulla parte di "Gestione DNS" ho:
-) 2 record di tipo "A" con nome pippo.com e contenuto due IP diversi;
-) 2 record di tipo "CNAME", uno con nome "www" (senza altro) e contenuto "pippo.com", uno con nome "_domainconnect" e contenuto "_domainconnect.gd.domaincontrol.com"
come dovrei modificare il tutto?

Grazie mille per la pazienza,
Diego
